### PR TITLE
Clean up override code with Optional.ifPresent

### PR DESCRIPTION
**Pull request by pgr0ss**
I think this is a more standard (and shorter) way to deal with Optionals. Let me know what you think.

### CODE REVIEW

File: src/main/java/me/dinowernli/grpc/polyglot/config/ConfigurationLoader.java [hunk 1]
```diff
@@ -100,49 +99,44 @@ private Configuration getNamedConfigurationInternal(String name) {
...
-        overrides.get().tlsClientKeyPath().get().toString());
-    }
-    if (overrides.get().tlsClientOverrideAuthority().isPresent()) {
-      resultBuilder.getCallConfigBuilder().setTlsClientOverrideAuthority(
-          overrides.get().tlsClientOverrideAuthority().get());
-    }
+      resultBuilder.getOutputConfigBuilder().setFilePath(path.toString());
+    });
+
+    resultBuilder.getProtoConfigBuilder().addAllIncludePaths(
```
Comment (dinowernli) [inline src/main/java/me/dinowernli/grpc/polyglot/config/ConfigurationLoader.java:117]: style nit: wrapped lines are indented by 4 (here and below)

**Review by dinowernli — CHANGES_REQUESTED** [review]
Thanks, looks good. Just a few minor style nits before we can merge this.

File: src/main/java/me/dinowernli/grpc/polyglot/config/ConfigurationLoader.java (hunk 1 above)
Comment (pgr0ss) [reply src/main/java/me/dinowernli/grpc/polyglot/config/ConfigurationLoader.java:117]: Do you have an IntelliJ code style I can import? If not, I can make the changes manually.

Comment (dinowernli) [reply src/main/java/me/dinowernli/grpc/polyglot/config/ConfigurationLoader.java:117]: Sadly, not yet. We should definitely add that eventually. For now, manually is the only way i'm afraid.

Comment (pgr0ss) [reply src/main/java/me/dinowernli/grpc/polyglot/config/ConfigurationLoader.java:117]: How does this look? https://github.com/grpc-ecosystem/polyglot/pull/67/commits/15e7e57b94e285624add70314ab6d792d462f731

**Review by dinowernli — APPROVED** [review]

**Comment by dinowernli** [comment]
Thanks for the contribution :)

**Pull request merged**